test/README.md

```diff
@@ -1,5 +1,63 @@
 ## End-to-End Testing
 
+### AWS E2E Testing
+Featured in this directory is a `./awsE2ESolutionSetup` script whose goal is to encapsulate deploying an E2E environment in AWS. This script will perform the following key actions:
+1. Deploy an Elasticsearch source cluster to EC2 using the following [CDK](https://github.com/lewijacn/opensearch-cluster-cdk/tree/migration-es)
+   * The CDK context options for this CDK can be used to customize this deployment and can be provided with a custom context file using the `--context-file` option
+   * A sample CDK context file can be seen [here](defaultCDKContext.json)
+2. Build the Docker Images
```

Comment:
For newbies, would be helpful to explain which Docker Images you're referring to here.  Mind add those details?

Reply:
Sure have added some more details now